buildkite/scripts/rosetta-integration-tests.sh

```diff
@@ -327,11 +237,15 @@ rosetta-cli configuration:validate ${ROSETTA_CONFIGURATION_FILE}
 echo "========================= ROSETTA CLI: CHECK:SPEC ==========================="
 rosetta-cli check:spec --all --configuration-file ${ROSETTA_CONFIGURATION_FILE}
 
-echo "========================= ROSETTA CLI: CHECK:CONSTRUCTION ==========================="
-rosetta-cli check:construction --configuration-file ${ROSETTA_CONFIGURATION_FILE}
+if [[ $MODE == "full" ]]; then
+
+  echo "========================= ROSETTA CLI: CHECK:CONSTRUCTION ==========================="
+  rosetta-cli check:construction --configuration-file ${ROSETTA_CONFIGURATION_FILE}
+
+  echo "========================= ROSETTA CLI: CHECK:DATA ==========================="
+  rosetta-cli check:data --configuration-file ${ROSETTA_CONFIGURATION_FILE}
 
-echo "========================= ROSETTA CLI: CHECK:DATA ==========================="
-rosetta-cli check:data --configuration-file ${ROSETTA_CONFIGURATION_FILE}
+  echo "========================= ROSETTA CLI: CHECK:PERF ==========================="
+  echo "rosetta-cli check:perf" # Will run this command when tests are fully implemented
 
-echo "========================= ROSETTA CLI: CHECK:PERF ==========================="
-echo "rosetta-cli check:perf" # Will run this command when tests are fully implemented
+fi
```

Comment:
The `check:spec` test is still called in the minimal version. I see some value in that, as it's faster and it can still catch up with errors like schema changes.

Reply:
Ah, you're right, `check:spec` is still being run. I missed that sorry. If it catches the schema changes, then indeed, maybe it's fine to run just that in CI.